hack/generate-install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
should we follow the knative release way to upload this yaml to the release artifact?